tcms/static/js/testcase_actions.js

```diff
@@ -755,6 +755,7 @@ function changeCasePriority(object_pk, value, callback) {
 
 function addCaseBug(form, callback) {
   var addBugInfo = Nitrate.Utils.formSerialize(form);
+//TODO: fix this
```

Comment:
Ignore this, it was a left over. https://github.com/Nitrate/Nitrate/pull/79/commits/6c27053a9f2e313e775132da270bcdccff5a9647